application/src/main/java/com/sanction/thunder/email/EmailService.java

```diff
@@ -0,0 +1,65 @@
+package com.sanction.thunder.email;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailService;
+import com.amazonaws.services.simpleemail.model.Body;
+import com.amazonaws.services.simpleemail.model.Content;
+import com.amazonaws.services.simpleemail.model.Destination;
+import com.amazonaws.services.simpleemail.model.Message;
+import com.amazonaws.services.simpleemail.model.SendEmailRequest;
+import com.sanction.thunder.models.Email;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EmailService {
+
+  private final AmazonSimpleEmailService emailService;
+
+  @Inject
+  public EmailService(AmazonSimpleEmailService emailService) {
+    this.emailService = emailService;
+  }
+
+  private static final Logger LOG = LoggerFactory.getLogger(EmailService.class);
+  private final String from = "noreply@sanctionco.com";
```

Comment:
Two things here:

1. Make the `from` variable static and change it to `FROM`: so that it looks like: `private static final String FROM = …`
2. Move it and the `LOG` above the constructor and above the `emailService` so that it's the first thing declared in the class.